vgscollect/src/main/java/com/verygoodsecurity/vgscollect/view/date/VGSDateFormat.kt

```diff
@@ -0,0 +1,112 @@
+package com.verygoodsecurity.vgscollect.view.date
+
+import java.text.SimpleDateFormat
+import java.util.*
+
+internal enum class VGSDateFormat(val format: String) {
+    mmyyyy("MM/yyyy"),
+    mmyy("MM/yy"),
+    mmddyyyy("MM/dd/yyyy"),
+    ddmmyyyy("dd/MM/yyyy"),
+    yyyymmdd("yyyy/MM/dd");
+
+    fun dateFromString(input: String?): Date? {
```

Comment:
@DonaldRG Should be moved to companion object?

Reply:
I don't think so, since this method need to use the format that is defined in the enum class